packages/next-app/src/utils/getTokenInfo.ts

```diff
@@ -0,0 +1,27 @@
+import tokensInfo from "public/tokens.json";
+
+function getTokenDictionary(json: any): {
```

Comment:
`#comment` JS doesn't have a Dictionary type... I think we've used `Mapping` most of the times, but would give it a more meaningful name e.g. `createTokenAddressInfoMapping`